spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/internal.scala

```diff
@@ -48,7 +47,6 @@ object internal extends Logging {
     dbEntity.setAttribute("locationUri", pathEntity)
     dbEntity.setAttribute("properties", dbDefinition.properties.asJava)
     dbEntity.setAttribute("owner", owner)
-    dbEntity.setAttribute("ownerType", "USER")
```

Comment:
I just removed the attribute because we haven't defined `ownerType` for spark db model.

Reply:
+1 for keeping this. IIRC,  Apache Hive SAC has this.